scripts/ci/libraries/_local_mounts.sh

```diff
@@ -54,7 +54,7 @@ function generate_local_mounts_list {
         "$prefix"setup.py:/opt/airflow/setup.py:cached
         "$prefix"tests:/opt/airflow/tests:cached
         "$prefix"kubernetes_tests:/opt/airflow/kubernetes_tests:cached
-        "$prefix"tmp:/opt/airflow/tmp:cached
```

Comment:
Oh ya this was wrong before. 👍

Reply:
Indeed. I wonder how it had happened that nobody noticed it so far :).